enterprise/servletjspapi/nbproject/project.properties

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 is.autoload=true
-release.external/servlet3.1-jsp2.3-api.jar=modules/ext/servlet3.1-jsp2.3-api.jar
+release.external/javax.servlet.jsp-api-2.3.3.jar=modules/ext/javax.servlet.jsp-api-2.3.3.jar
```

Comment:
Please consider if a mapping:
```
release.external/javax.servlet.jsp-api-2.3.3.jar=modules/ext/servlet3.1-jsp2.3-api.jar
```
would be better. That way there would be no external change necessary. You could consider the file name a kind of API and by retaining the original name, you keep the API stable.

That way several of the other changes here are not necessary.

Reply:
Matthias, Thanks for the review. This is indeed a better approach to handle this. I have made the necessary changes in the latest commit.